doc/classes/XRHandTracker.xml

```diff
@@ -0,0 +1,223 @@
+			<description>
+				Returns the linear velocity for the given hand joint.
+			</description>
+		</method>
+		<method name="get_hand_joint_radius" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="joint" type="int" enum="XRHandTracker.HandJoint" />
+			<description>
+				Returns the radius of the given hand joint.
+			</description>
+		</method>
+		<method name="get_hand_joint_transform" qualifiers="const">
+			<return type="Transform3D" />
+			<param index="0" name="joint" type="int" enum="XRHandTracker.HandJoint" />
+			<description>
+				Returns the transform for the given hand joint.
+			</description>
+		</method>
+		<method name="set_hand_joint_angular_velocity">
+			<return type="void" />
+			<param index="0" name="joint" type="int" enum="XRHandTracker.HandJoint" />
+			<param index="1" name="angular_velocity" type="Vector3" />
+			<description>
+				Sets the angular velocity for the given hand joint.
+			</description>
+		</method>
+		<method name="set_hand_joint_flags">
+			<return type="void" />
+			<param index="0" name="joint" type="int" enum="XRHandTracker.HandJoint" />
+			<param index="1" name="flags" type="int" enum="XRHandTracker.HandJointFlags" is_bitfield="true" />
+			<description>
+				Sets flags about the validity of the tracking data for the given hand joint.
+			</description>
+		</method>
+		<method name="set_hand_joint_linear_velocity">
+			<return type="void" />
+			<param index="0" name="joint" type="int" enum="XRHandTracker.HandJoint" />
+			<param index="1" name="linear_velocity" type="Vector3" />
+			<description>
+				Sets the linear velocity for the given hand joint.
+			</description>
+		</method>
+		<method name="set_hand_joint_radius">
+			<return type="void" />
+			<param index="0" name="joint" type="int" enum="XRHandTracker.HandJoint" />
+			<param index="1" name="radius" type="float" />
+			<description>
+				Sets the radius of the given hand joint.
+			</description>
+		</method>
+		<method name="set_hand_joint_transform">
+			<return type="void" />
+			<param index="0" name="joint" type="int" enum="XRHandTracker.HandJoint" />
+			<param index="1" name="transform" type="Transform3D" />
+			<description>
+				Sets the transform for the given hand joint.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="hand" type="int" setter="set_hand" getter="get_hand" enum="XRHandTracker.Hand" default="0">
+			The type of hand.
+		</member>
+		<member name="hand_tracking_source" type="int" setter="set_hand_tracking_source" getter="get_hand_tracking_source" enum="XRHandTracker.HandTrackingSource" default="0">
+			The source of the hand tracking data.
+		</member>
+		<member name="has_tracking_data" type="bool" setter="set_has_tracking_data" getter="get_has_tracking_data" default="false">
+			If [code]true[/code], the hand tracking data is valid.
+		</member>
+	</members>
+	<constants>
+		<constant name="HAND_LEFT" value="0" enum="Hand">
+			A left hand.
```

Comment:
Hm. Well, before this PR we have two left/right hand enums:

- `XRPositionalTracker.TrackerHand`
- `OpenXRInterface.Hand`

The 2nd of those is going to be deprecated after this PR, so I think we can mostly ignore it.

The 1st of those has different values: unknown (0), left (1) and right (2). That's a little inconvenient for hand tracking because their is no unknown and we can use left (0) and right (1) as array indexes (and the max value for use in loops). 

Given that, I feel like it's OK for `XRHandTracker` to have a hand enum that fits its purposes. And, even if we decided to consolidate to a single enum on `XRServer`, we can't change `XRPositionalTracker` to use it for backwards compatibility reasons. Maybe when we can break compatibility (like, Godot 5) we could consolidate on a single enum in `XRServer`? Although, given the different needs of the two enums, I personally still think it'd be OK to have the two of them.